lib/public/components/common/popover/overflowBalloon.js

```diff
@@ -53,16 +53,33 @@ export const overflowBalloon = (content, options = null) => {
              */
             onTriggerNodeChange: function (previousTriggerNode, newTriggerNode) {
                 if (previousTriggerNode) {
-                    const node = stretch ? previousTriggerNode.parentNode : previousTriggerNode;
-
-                    node.removeEventListener('mouseover', this.showPopover);
-                    node.removeEventListener('mouseleave', this.hidePopover);
+                    previousTriggerNode.removeEventListener('mouseover', this.prevousNodeMouseoverCallback);
+                    previousTriggerNode.removeEventListener('mouseleave', this.hidePopover);
                 }
 
                 if (newTriggerNode) {
                     const node = stretch ? newTriggerNode.parentNode : newTriggerNode;
+                    const key = newTriggerNode.getAttribute('data-popover-key');
+
+                    const newNodeMouseoverCallback = () => {
+                        this.showPopover();
+                        const selector = `.popover[data-popover-key="${key}"]`;
+                        const contentNode = document.querySelector(selector);
+
+                        contentNode.addEventListener('mouseover', () => {
```

Comment:
Looking at the JS MDN docs, looks like the method will add to a list of eventlisteners and not replace them by name.
Should it not be better to first attempt to remove existing listeners and then add them? Currently when I logged locally, the listeners are chained and called tens of times because of this